go/vt/vtgate/planbuilder/operators/system_tables.go

```diff
@@ -122,7 +121,7 @@ func isTableNameCol(col *sqlparser.ColName) bool {
 type notImplementedSchemaInfoConverter struct{}
 
 func (f *notImplementedSchemaInfoConverter) ColumnLookup(*sqlparser.ColName) (int, error) {
-	return 0, vterrors.Errorf(vtrpcpb.Code_UNIMPLEMENTED, "Comparing table schema name with a column name not yet supported")
+	return 0, vterrors.VT12001("comparing table schema name with a column name is not yet supported")
```

Comment:
```suggestion
	return 0, vterrors.VT12001("comparing table schema name with a column name")
```

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)